sdk/core/azure-core/inc/azure/core/http/policy.hpp

```diff
@@ -15,11 +15,6 @@
 #include "azure/core/logging/logging.hpp"
 #include "azure/core/uuid.hpp"
 
-// Need to include the transport adapters offered by the SDK here so anyone can use them by just
-// including policy.hpp
-#include "azure/core/http/curl/curl.hpp"
-#include "azure/core/http/winhttp/win_http_client.hpp"
```

Comment:
@antkmsft why can we remove this now? @vhvb1989, does the initial reason these were added (from the comments) no longer hold true?

Reply:
Per our offline discussion, including Victor as well, this is the right move.